samples/createJobFromPreset.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2020, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(
+  projectId = process.env.GOOGLE_CLOUD_PROJECT,
+  location,
+  inputUri,
+  outputUri,
+  preset
+) {
+  // [START transcoder_create_job_from_preset]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // projectId = 'my-project-id';
+  // location = 'us-central1';
+  // inputUri = 'gs://my-bucket/my-video-file';
+  // outputUri = 'gs://my-bucket/my-output-folder/';
+  // preset = 'preset/web-hd';
+
+  // Imports the Transcoder library
+  const {TranscoderServiceClient} = require('@google-cloud/video-transcoder');
+
+  // Instantiates a client
+  const transcoderServiceClient = new TranscoderServiceClient();
+
+  async function createJobFromPreset() {
+    // Construct request
+    const request = {
+      parent: transcoderServiceClient.locationPath(projectId, location),
+      job: {
+        inputUri: inputUri,
+        outputUri: outputUri,
+        templateId: preset,
+      },
+    };
+
+    // Run request
+    const [response] = await transcoderServiceClient.createJob(request);
+    console.log(`Job: ${response.name}`);
+  }
+
+  createJobFromPreset();
+  // [END transcoder_create_job_from_preset]
+}
+
+// node createJobFromPreset.js <projectId> <location> <inputUri> <outputUri> <preset>
+main(...process.argv.slice(2));
```

Comment:
would you mind ending these with

```
process.on('unhandledRejection', (err) => {
  console.error(err.message);
  process.exitCode = 1;
});
main(...process.argv.slice(2));
```

so that we catch any errors that throw inside the async func?

Reply:
Done